src/main/java/pl/byczazagroda/trackexpensesappbackend/service/FinancialTransactionServiceImpl.java

```diff
@@ -28,8 +29,8 @@ public class FinancialTransactionServiceImpl implements FinancialTransactionServ
 
     private final FinancialTransactionRepository financialTransactionRepository;
     private final FinancialTransactionModelMapper financialTransactionModelMapper;
+    //tu jest lipa WalletService powinnien to obsługiwać
```

Comment:
Please remove this comment. If you have question or some comments please add it on Discord group (os-backend). You can also write directly to leaders.

Reply:
And as a last resort, if you give a comment in the code, it is a request to use English. This is because you usually work in multinational teams. And not every developer speaks Polish.